pythonforandroid/recipes/libffi/__init__.py

```diff
@@ -37,46 +36,10 @@ def build_arch(self, arch):
                     '--prefix=' + self.get_build_dir(arch.arch),
                     '--disable-builddir',
                     '--enable-shared', _env=env)
-            # '--with-sysroot={}'.format(self.ctx.ndk_platform),
-            # '--target={}'.format(arch.toolchain_prefix),
 
-            # ndk 15 introduces unified headers required --sysroot and
-            # -isysroot for libraries and headers. libtool's head explodes
-            # trying to weave them into it's own magic. The result is a link
-            # failure trying to link libc. We call make to compile the bits
-            # and manually link...
+            shprint(sh.make, '-j5', 'libffi.la', _env=env)
```

Comment:
since you touch this line...could you remove the harcdoded cpu count in make command?

I suggest this (we use that in some other recipes):

```
shprint(sh.make, '-j', str(cpu_count()), 'libffi.la', _env=env)
```

**Note:** also you should import cpu_count: `from multiprocessing import cpu_count`